api/jobs/params/update-job-parameters.ts

```diff
@@ -1,4 +1,64 @@
-import { CreateUpdateJobBaseParameters } from "./create-update-job-base-parameters";
+import { BaseParameters } from "../../parameters";
+import { SmartlingException } from "../../exception";
+import { CallbackMethod } from "./callback-method";
 
-export class UpdateJobParameters extends CreateUpdateJobBaseParameters<UpdateJobParameters> {
+interface CustomFieldBasicRequest {
+    fieldUid: string,
+    fieldValue: string
+}
+
+export class UpdateJobParameters extends BaseParameters {
+    setName(jobName: string): this {
```

Comment:
I also didn't get why this PR needs changes in completely different API client's code

Reply:
> looks like github feels there were too many changes.

what does this mean? we decided to change something in the code only because of some UI in gh showing something in a wrong way?

Reply:
I've introduced this change while trying to avoid creating a new parameters file. After I've determined that the hierarchy of the existing parameters makes this impossible, and a new file is inevitable, I've decided not to revert the changes. Renaming the file caused github to show it as a new file, as it contained too many changes.

Reply:
let's revert this change. has nothing to do with this pr. I'm ok to merge it after reverting this

Reply:
Done